api/models/blocks/index.js

```diff
@@ -59,9 +59,37 @@ const get = async (blockId) => {
   }
 };
 
+const update = async (blockId, requestBody) => {
+  let payload = {};
+  
+  // copy valid user fields from request body
+  Object.keys(requestBody).forEach(key => {
```

Comment:
i actually learned that knex will ignore any null values you pass to the update method.  so it might be preferable to just do something like:
```
const payload = {
  name: req.body.name,
  language: req.body.language,
};
```
i know i didn't do this for users originally, so it would make sense to update in both places if you think it's a good change.